app/questionnaire/rules.py

```diff
@@ -93,6 +102,13 @@ def evaluate_condition(condition, answer_value, match_value):
     return match_function(answer_value, match_value)
 
 
+def casefold(value):
```

Comment:
Is this because of the other languages?

Reply:
Yes - casefold takes into account non ASCII char codes